framework/include/mesh/MooseMesh.h

```diff
@@ -735,10 +735,11 @@ class MooseMesh : public MooseObject, public Restartable, public PerfGraphInterf
   /**
    * Get the associated subdomainIDs for the subdomain names that are passed in.
    *
-   * @param subdomain_name The names of the subdomains
+   * @param subdomain_names The names of the subdomains
    * @return The subdomain ids from the passed subdomain names.
    */
   std::vector<SubdomainID> getSubdomainIDs(const std::vector<SubdomainName> & subdomain_name) const;
+  std::set<SubdomainID> getSubdomainIDs(const std::set<SubdomainName> & subdomain_name) const;
```

Comment:
```suggestion
  std::vector<SubdomainID> getSubdomainIDs(const std::vector<SubdomainName> & subdomain_names) const;
  std::set<SubdomainID> getSubdomainIDs(const std::set<SubdomainName> & subdomain_names) const;
```

Reply:
auto-format add another line if we do it to the vector